### ISSUE TITLE
[Turbo] Support for broadcasting page refreshes

### ISSUE THREAD

**Issue by seb-jean** [symfony/ux#1870]
Hi,

Turbo 8 added a feature: [Broadcasting page refreshes](https://turbo.hotwired.dev/handbook/page_refreshes#broadcasting-page-refreshes). Currently, it is unfortunately not possible to use it with Symfony. This will greatly simplify the code, and you won't have to worry so much about `<turbo-stream>`.

I put this article to accompany this issue: https://dev.37signals.com/page-refreshes-with-morphing-demo/

What do you think ?

**Comment by Xbirdfr** [comment]
Hi ! i'm also looking for this feature with Symfony

actually, it seem working by updating from importmap, but I don't know if something else broke

```php 
// /importmap.php
'@hotwired/turbo' => [
        'version' => '8.0.12',
    ],
```

**Comment by Xbirdfr** [comment]
Hi ! i'm also looking for this feature with Symfony

do you have a workaround until the feature is added to S.Turbo ?

**Comment by smnandre** [comment]
TurboStream helpers are available in Symfony UX Turbo 2.21
